plugin/gcp_config.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/authmetadata"
```

Comment:
nit: import ordering? I think `goimports` fixes this

Reply:
Fixed the imports but I won't link the commit since I'm going to squash them all before I mark this ready as a PR.